news/2024/2024-05-20-the-followpoint-yesmydarknesss-the-osu-catch-prodigy.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "The Followpoint: YesMyDarknesss, the osu!catch Prodigy"
+date: 2024-05-20 11:00:00 +0000
+---
+
+Read up on the latest interview with YesMyDarknesss, the osu!catch #1 player.
```

Comment:
It would be nice to mention they hold #1 longer than any other current #1, but I don't know how to word it without making a wall of text